src/modules/import/domain/import-event-handler.ts

```diff
@@ -64,9 +67,37 @@ export class ImportEventHandler {
             return this.logger.error(`No import data itemns found for Importvorgang:${importvorgangId}`);
         }
 
+        /* eslint-disable no-await-in-loop */
```

Comment:
`/* eslint-disable no-await-in-loop */` disables the rule for *every* line after this.
Please place individual `// eslint-disable-next-line no-await-in-loop` above the offending lines